proselint/checks/__init__.py

```diff
@@ -39,26 +40,28 @@ def get_checks(options: dict) -> list[Callable[[str, str], list[ResultCheck]]]:
     # TODO: benchmark consecutive runs of this
     #       config should only translate once to check-list
     checks = []
-    check_names = [key for (key, val) in options["checks"].items() if val]
+    check_names = ["." + key for (key, val) in options["checks"].items() if val]
```

Comment:
I'll undo this when I get to university. If you'd like me to do the rebases for you, since I also did this one, I'll refrain from making post-rebase changes.

Reply:
I somehow was forced to rebase again on my main (was there an evil force-push?) and already changed it back